### PR TITLE
Revives Shuttle Navigation Computers to all syndicate shuttles + Vox/ERT

### PR DESCRIPTION
**Pull request by datlo**
This PR is a deconflicted #10522. I've also fixed a couple bugs and changed a couple things here and there, mostly cutting out the admin shuttle nav computer to keep things straight and simple.

Huge thanks to Darkmight9 for finding the solution to nav consoles being able to see inside the station
Also thanks to Ionward for spriting the console screens

-----------

Add a shuttle navigation computer ported from tg to : 
- The Nuke Ops shuttle
- The SIT and SST shuttle
- The Vox Skipjack
- The ERT shuttle (restricted to station only)

These computers allow you to set a custom destination to go to. Pictures below for details

Additionally, most offstation shuttle consoles are now impossible to destroy/deconstruct.

The ERT shuttle now has atmos blocking fans (since it can dock in space) under its airlocks, and also has a security camera console.

navigation1

Once you use the computer, you enter a remote view of the area you're going to set as landing
navigation2

You can then select any area that is 1. Space turfs only 2. Not used by another shuttle as a docking port 3. Not otherwise restricted (syndicate depot)

navigation3

You can then use that destination using the shuttle console, as you'd usually move

navigation4

And you're there!

navigation5-5

If you try to select any area you can't go to, the relevant turfs will be highlighted in red.

navigation5

You can also choose another zlevel to move to. Syndicate shuttles cannot go to the derelict. (no NT installations there + balance concern with the depot being in that zlevel)

navigation6

navconsole7

🆑
add: Added navigation computers to the nuclear, SIT, SST, ERT, and vox shuttles. This computer lets you set a custom destination to jump to.
tweak: Most off-station shuttle consoles are now indestructible and impossible to deconstruct.
add: Added atmos blocking tiny fans and a security camera console to the ERT shuttle.
/🆑

### CODE REVIEW

**Comment by datlo** [comment]
Added sprites by Ionward
navconsole
syndinavconsole

Fixed navconsoles being able to see inside the station, you're now restricted to space tiles. You can still outside the station and through windows but... really not that big a deal, imo. It looks fine.

PR is ready to be merged once the golem PR is merged and I add the map changes to it

**Comment by Shadeykins** [comment]
@datlo Marry me?

**Comment by Quantum-M** [comment]
Finally, security can now fine the syndicate for illegal/improper parallel parking.

**Comment by datlo** [comment]
I think the PR is ready but I still need to do a final round of testing/reviewing tomorrow
I'll remove the WIP once it's all clear

File: code/modules/shuttle/shuttle_rotate.dm [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+//prevents shuttles attempting to rotate this since it messes up sprites
+/obj/structure/table/shuttleRotate(rotation, params)
+	params = NONE
+	return ..()
+
+/obj/structure/table_frame/shuttleRotate(rotation, params)
+	params = NONE
+	return ..()
+
+/************************************Machine rotate procs************************************/
```
Comment (variableundefined) [inline code/modules/shuttle/shuttle_rotate.dm]: No leftover code please

File: code/_globalvars/lists/objects.dm [hunk 2]
```diff
@@ -14,6 +14,7 @@ GLOBAL_LIST_INIT(janitorial_equipment, list())		//list of janitorial equipment
 GLOBAL_LIST_INIT(crafting_recipes, list()) //list of all crafting recipes
 GLOBAL_LIST_INIT(prisoncomputer_list, list())
 GLOBAL_LIST_INIT(cell_logs, list())
+GLOBAL_LIST_INIT(navigation_computers, list())
```
[the remark this answers is not in the document]
Comment (variableundefined) [reply code/_globalvars/lists/objects.dm]: No need to take action on above comment.

**Comment by datlo** [comment]
Leftover code removed

**Comment by Kyep** [comment]
I ended up playing as a SIT the other day.
I have to say, this system seems to have made the SIT shuttle harder to park.
SIT don't get hardsuits by default. They're meant to be able to dock right up against the hull, so they can just open a door and cut in, without suits.

Under this system, I found that the only place I could position a SIT shuttle to get suit-less access to the station was on top of the arrivals pods. 

Its almost as if you can't place the shuttle anywhere that touches the station... which is exactly what the SIT shuttle needs to do to dock properly.

**Comment by datlo** [comment]
Unless I made a mistake on that the SIT still has the option of using their regular docks to park next to the station as they could before. This is just an additional option if youd rather dock in space and buy a suit.

**Pull request merged**